### ISSUE TITLE
[Bug Report] form-item with slotted content is missing while the label prop is not set.

### ISSUE THREAD

**Issue by gitpaladin** [ElemeFE/element#5895]
### Element UI version
1.3.7

### OS/Browsers version
Win10/Chrome59

### Vue version
2.4.1

### Reproduction Link
[https://jsfiddle.net/33u04mg4/3/](https://jsfiddle.net/33u04mg4/3/)

### Steps to reproduce
1. unset the label prop of the el-form-item.
2. using the label slot to render the label content.

### What is Expected?
Using the named slot label content to render the label

### What is actually happening?
The label of the form item is blank.

**Comment by gitpaladin** [comment]
The `form-item.vue` with the template have a `v-if` condition.

```HTML
    <label :for="prop" class="el-form-item__label" v-bind:style="labelStyle" v-if="label">
      <slot name="label">{{label + form.labelSuffix}}</slot>
    </label>
``` 

But the docuemnt doesn't say the `label` prop will affact the slot content rendering.

**Comment by freeman-g** [comment]
I also noticed that the label slot only works if the label prop is set to something at the form item level.

```
          <el-form-item label="placeholder">
            <label slot="label">
              Version <el-tooltip effect="dark" content="My tooltip text" placement="top">
                        <i class="el-icon-information"></i>
                      </el-tooltip>
            </label>
            <input type="text"></input>
            </Choices>
          </el-form-item>
```

**Comment by Leopoldthecoder** [comment]
Fixed in #5921
